third_party/openthread/src/ncp/spinel.h

```diff
@@ -979,6 +979,9 @@ typedef enum
      */
     SPINEL_PROP_IPV6_ICMP_PING_OFFLOAD  = SPINEL_PROP_IPV6__BEGIN + 5, ///< [b]
 
+    SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE
```

Comment:
Wasn't there supposed to be two tables, one read-only with all multicast addresses, and one that is read/write for application-added multicast addresses? Or am I misremembering an earlier discussion?